### PR TITLE
Update README.md with some stuff that helped me install.

### PR DESCRIPTION
**Pull request by benschac**
Got me up and running :) Some stuff that's super helpful to document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Prerequisites
 Getting Started
 ---------------
 
-The easiest way to get started is to clone the repository:
+The easiest way to get started is to clone the repository: (NOTE: node version 4.2.2 and npm version 2.X.X is needed as a minumium requirement)
```
Comment (SaintPeter) [inline README.md]: > (NOTE: node version 4.2.2 and npm version 2.X.X ~~is~~ **are** needed as a minumium requirement)

Should be `are`

`minumium` is spelled `minimum`

File: README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ bower install
 # Create a .env file and populate it with the necessary API keys and secrets:
 touch .env
 ```
-
+(Note if you're having issues with any of these steps or have installed something with npm delete your modules AND clean your npm cache with npm cache clear)
```
Comment (SaintPeter) [inline README.md]: > (Note if you're having issues with any of these steps or have installed something with npm **COMMA HERE** delete your modules AND clean your npm cache with `npm cache clear`)

Also added back ticks around "`npm cache clear`"

Also, I'm not really sure what you mean by "installed something with npm".

**Comment by SaintPeter** [comment]
After you make these corrections, please squash your commits and force push the branch back up.

**Comment by SaehanLee** [comment]
I'm trying to get up and running but I don't know how to edit the .env file. Is there some instruction on how to do that?

**Comment by ltegman** [comment]
@SaehanLee You'll need to create the file and can then just open and edit it with a text editor. Visit [Help Conbtributors](https://gitter.im/FreeCodeCamp/HelpContributors) if you need more assistance.

**Comment by BerkeleyTrue** [comment]
closing as stale. Please feel free to re submit when ready.

**Pull request closed without merging**